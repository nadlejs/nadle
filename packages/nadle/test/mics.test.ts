import { it, describe } from "vitest";
import { exec, expectPass } from "setup";

describe("when not given any tasks", () => {
	it("show all available tasks", async () => {
		await expectPass(exec``);
	});
});
