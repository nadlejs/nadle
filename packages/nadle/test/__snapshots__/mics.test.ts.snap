// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`when not given any tasks > show all available tasks 1`] = `
[log] No tasks were specified. Please specify one or more tasks to execute, or use the --list option to view available tasks.
[log] 
<Bold><Green>RUN SUCCESSFUL</Green></BoldDim> in <Bold>_ms</BoldDim> <Dim>(<Bold>0</BoldDim><Dim> task executed)</BoldDim>
`;
